integrations/hexo-ariada/tests/hexo-ariada.test.js

```
@@ -0,0 +1,153 @@
+// SPDX-FileCopyrightText: 2026 Agonist Development AB
+// SPDX-License-Identifier: EUPL-1.2
+'use strict';
+
+const assert = require('node:assert/strict');
```

ESLint

[error] A `require()` style import is forbidden.

(`@typescript-eslint/no-require-imports`)

---

[error] 'require' is not defined.

(no-undef)